_pages/user-experience/agency-logo.md

```diff
@@ -32,17 +32,21 @@ Your agency logo will appear to the right of the Login.gov logo in your integrat
 <div class="grid-row">
   <div class="float-left agency-logo-width">
     <img src="{{ site.baseurl }}/assets/img/do_logo.svg" alt="An example agency logo in black on a white background" class="display-block green-bottom-border">
-      <div class="text-green float-left">
-        <svg role="img" class="height-3 usa-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z"/></svg>
-      </div>
-      <div class="margin-left-3">
-        <span class="text-uppercase text-bold text-green">Do</span>: Add a logo that is high contrast on a transparent background.
-      </div>
+    <div class="text-green float-left">
+      <svg role="img" class="height-3 usa-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+        <use xlink:href="{{site.baseurl}}/assets/img/sprite.svg#check_circle"></use>
+      </svg>
+    </div>
```

Comment:
Is it not possible to use the `icon.html` include here, and below?

**Edit:** Guessing it's the `height-3` that makes it challenging?

Reply:
that was why i hadn't used it. but the `floats` were annoying me so i replaced it with the [image-example code](https://github.com/18F/identity-design-system/blob/main/docs/_components/image-example.md?plain=1) you pointed me towards which i was able to make work with icon_list.html